Benqi-Agents/Large-Mint-Burn/README.md

```diff
@@ -0,0 +1,43 @@
+# Large PGL Mint-Burn Agent
+
+## Description
+
+This agent detects Mints and Burns on PGL contract with large QI-WAVAX amounts.
+
+> Large is defined as a percent of the token reserve.
+
+## Supported Chains
+
+- Avalanche
+
+## Alerts
+
+- BENQI-8-1
+
+  - Fired when a `Mint` event is emitted on PGL contract with a large QI or WAVAX amount.
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata includes:
+    - `source`: address of the user initializing the Mint.
+    - `amount0`: amount of `QI` tokens that were Minted.
+    - `amount1`: amount of `WAVAX` tokens that were Minted.
+
+- BENQI-8-2
+
+  - Fired when a `Burn` event is emitted on PGL contract with a large QI or WAVAX amount.
+  - Severity is always set to "Info"
+  - Type is always set to "Info"
+  - Metadata includes:
+    - `source`: address of the user initializing the Burn.
+    - `amount0`: amount of `QI` tokens that were Burned.
+    - `amount1`: amount of `WAVAX` tokens that were Burned.
+    - `to`: address to which the tokens were transfered.
+
+## Test Data
+
+There's no txs including large Mints/Burns on PGL contract. We have deployed a PoC contract to emit the monitored events. The agent behaviour can be verified with the following test transactions (Ropsten):
+> note that `TESTNET_PGL_CONTRACT` should be used instead of `PGL_CONTRACT`.
```

Comment:
We should mention where and how this should be changed in-case there are some people using this agent who aren't too familiar with Forta.

Reply:
Something simple like mentioning to change the inputs to `provideHandleTransaction` in the default exports in `agent.ts` should suffice.